src/devices/cpu/tms34010/34010ops.hxx

```diff
@@ -2245,7 +2246,34 @@ void tms34020_device::cvdxyl_a(uint16_t op)
 
 void tms34020_device::cvdxyl_b(uint16_t op)
 {
-	logerror("020:cvdxyl_b\n");
+	// Convert destination XY address to linear
+	// (Y half of Rd x DPTCH) + (X half of Rd x PSIZE) + (A4 or B4) ->  Rd
+	XY *rd = &BREG_XY(DSTREG(op));
+	off_t dptch = DPTCH();
+	off_t offset = BREG(4);
+
+	off_t result = (rd->y * dptch) + (rd->x * (1 << m_pixelshift)) + offset;
```

Comment:
Can these things be `const`?  I don’t think any of them should be modified after assignment.